sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/models.py

```diff
@@ -15,11 +15,11 @@
     from ._shared._generated.v7_0 import models as _models
 
 
-class SecretAttributes(object):
+class SecretProperties(object):
     """A secret's id and attributes."""
 
     def __init__(self, attributes, vault_id, **kwargs):
-        # type: (_models.SecretAttributes, str, **Any) -> None
+        # type: (_models.SecretProperties, str, **Any) -> None
```

Comment:
Has the generated class's name changed?

Reply:
thanks, I got a little too trigger happy